Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -48,13 +48,18 @@
 param = demisto.params()
 TIME_FIELD = param.get('fetch_time_field', '')
 FETCH_INDEX = param.get('fetch_index', '')
-FETCH_QUERY = param.get('fetch_query', '')
+FETCH_QUERY_PARM = param.get('fetch_query', '')
+RAW_QUERY = param.get('raw_query', '')
 FETCH_TIME = param.get('fetch_time', '3 days')
 FETCH_SIZE = int(param.get('fetch_size', 50))
 INSECURE = not param.get('insecure', False)
 TIME_METHOD = param.get('time_method', 'Simple-Date')
 TIMEOUT = int(param.get('timeout') or 60)
 MAP_LABELS = param.get('map_labels', True)
+TIME_RANGE_START = param.get('time_range_start')
+TIME_RANGE_END = param.get('time_range_end')
+
+FETCH_QUERY = RAW_QUERY or FETCH_QUERY_PARM
 
 
 def get_timestamp_first_fetch(last_fetch):
```

Comment:
Let's rename this function as the usability has changed and we use this for several time-casting